pkg/server/commands/list_objects.go

```diff
@@ -300,7 +301,7 @@ func (q *ListObjectsQuery) evaluate(
 				User:             sourceUserRef,
 				ContextualTuples: req.GetContextualTuples().GetTupleKeys(),
 				Context:          req.GetContext(),
-			}, reverseExpandResultsChan, reverseExpandResolutionMetadata)
+			}, reverseExpandResultsChan, reverseExpandResolutionMetadata, storage.ConsistencyOptions{Preference: req.GetConsistency()})
```

Comment:
why not make `consistency` part of `ReverseExpandRequest`? it would be more readable

Reply:
I think that's a good call as it's request data realistically

Reply:
Done in c3c6d511e255f424d3c85e7a61f9e376cbc5f478